repos/spack_repo/builtin/packages/csdp/package.py

```diff
@@ -15,15 +15,26 @@ class Csdp(MakefilePackage):
     homepage = "https://projects.coin-or.org/Csdp"
     url = "https://www.coin-or.org/download/source/Csdp/Csdp-6.1.1.tgz"
 
-    license("CPL-1.0")
+    license("EPL-2.0", checked_by="d-torrance")
 
+    version("6.2.0", sha256="7f202a15f33483ee205dcfbd0573fdbd74911604bb739a04f8baa35f8a055c5b")
     version("6.1.1", sha256="0558a46ac534e846bf866b76a9a44e8a854d84558efa50988ffc092f99a138b9")
 
-    depends_on("c", type="build")  # generated
+    depends_on("c", type="build")
 
-    depends_on("atlas")
+    depends_on("blas")
+    depends_on("lapack")
 
     def edit(self, spec, prefix):
         mkdirp(prefix.bin)
         makefile = FileFilter("Makefile")
         makefile.filter("/usr/local/bin", prefix.bin)
+
+    @property
+    def build_targets(self):
+        return [
+            "LIBS=-L../lib -lsdp "
+            f"{self.spec['blas'].libs.ld_flags} "
+            f"{self.spec['lapack'].libs.ld_flags} "
+            "-lm"
+        ]
```

Comment:
These are not build targets. We should use [flag_handler](https://spack.readthedocs.io/en/latest/packaging_guide_build.html#compiler-flags) for these kind of things.